_rfcs/rfc-0002.md

```diff
@@ -28,11 +28,13 @@ This process will be reviewed with a plan to become an official process in Augus
 
 # Motivation
 
-Put simply, as a society we want to make as big a difference as possible in the lives of Edmonton developers and the broader community. Working Groups allow all Dev Edmonton Society members to quickly and effectively perform interesting work of their choosing.
+As a society we want to improve the lives of Edmonton developers as much as possible, enriching the broader community through them. Working Groups allow all Dev Edmonton Society members to quickly and effectively perform interesting work of their choosing.
 
-As well Working Groups keep the work of DES open, collaborative, and welcoming to members of all abilities, interest, and availability.
+Working Groups keep the work of DES open, collaborative, and welcoming to members of all abilities, interest, and availability.
 
-DES provides these Working Groups with resources, a platform, and shared experience to make this possible then steps back to allow members to do the work.
+DES provides Working Groups with resources, a platform, and shared experience to empower members.
+
+In the past the Board, along with a small adhoc group of members, has done most of the day-to-day work of the Society. The Working Groups are succeeding if the majority of the Society's work leaves the Board and is instead taken up by the Working Groups.
```

Comment:
@jfeaver thoughts? Does this make the motivation clearer?

Reply:
Yeah, this is great.